Tasks/DownloadFileShareArtifactsV0/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "downloadfileshareartifacts",
+  "version": "0.1.0",
+  "description": "Download FileShare Artifacts Task",
```

Comment:
good to replace FileShare with Fileshare across all files to be consistent

Reply:
done